README.md

```diff
@@ -40,12 +40,17 @@ label1:
 
 From a boolean logic perspective, top-level match objects are `OR`-ed together and individual match rules within an object are `AND`-ed. Combined with `!` negation, you can write complex matching rules.
 
+> ⚠️ This action uses [micromatch](https://www.npmjs.com/package/micromatch) to apply glob patterns.
```

Comment:
@kachkaev, one last thing before merging. This action actually uses [minimatch](https://www.npmjs.com/package/minimatch). Could you please change the README.md?

Reply:
You are right, well-spotted! I addressed this in https://github.com/actions/labeler/pull/316/commits/d40596e5db2302f8bd180870610f0059a6e3ace2